src/components/shared/ArrowPanel.js

```diff
@@ -4,15 +4,20 @@
 
 // @flow
 
+// This implements a panel with a "arrow" triangle graphic that points to the
+// button that triggers it.
+// Please do not use this component directly. This is used by ButtonWithPanel
```

Comment:
Is there any reason not to directly include it in the other file now that it's private?

Reply:
I wanted to keep the files shorter. I believe that using an index that exports only the public API makes it obvious enough.